File: Modul-4/C/PRAK403-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int angka1, angka2;

  printf("Input\n");
  scanf("%d %d", &angka1, &angka2);

  printf("Output\n");
  if (angka1 < angka2)
  {
    for (int i = angka1; i <= angka2; i++)
    {
      printf("%d %d", i, angka2 - (i - angka1));
      if (i != angka2)
      {
        printf(" - ");
      }
    }
  }
  else
  {
    for (int i = angka1; i >= angka2; i--)
    {
      printf("%d %d", i, angka2 + (angka1 - i));
      if (i != angka2)
      {
        printf(" - ");
      }
    }
  }

  printf("\n");

  return 0;
}

File: Modul-4/Python/PRAK403-2310817110006-Adrian_Bintang_Saputera.py
angka1, angka2 = map(int, input("Input\n").split())

print("Output")
if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        print(f"{i} {angka2 - (i - angka1)}", end="")
        if i != angka2:
            print(" - ", end="")
else:
    for i in range(angka1, angka2 - 1, -1):
        print(f"{i} {angka2 + (angka1 - i)}", end="")
        if i != angka2:
            print(" - ", end="")

print()